docker/Dockerfile

```diff
@@ -23,14 +23,8 @@ RUN apt-get update && apt-get install -y openjdk-8-jdk
 RUN wget https://dl.google.com/android/repository/sdk-tools-linux-4333796.zip
 RUN unzip sdk-tools-linux-4333796.zip -d /opt/android/sdk
 RUN yes | $ANDROID_HOME/tools/bin/sdkmanager --licenses
-RUN $ANDROID_HOME/tools/bin/sdkmanager "build-tools;26.0.2" "platform-tools" "platforms;android-26" --sdk_root=/opt/android/sdk
-ENV PATH /opt/android/sdk/tools:/opt/android/sdk/platform-tools:$PATH
-
-# Install CMake 3.6
-WORKDIR /opt
```

Comment:
Removal of that line changed the working directory at the end of the build (initial directory when running the container) from `/opt` to `/opt/android/sdk` as set in line 10. I suggest to set the `WORKDIR` to `/opt/ros_android` explicitly at the end and to also set the default command, e.g.

```
VOLUME /opt/ros_android
WORKDIR /opt/ros_android
CMD ['bash']
```
for usage without [run.sh](https://github.com/Intermodalics/ros_android/blob/2c755cd743293d315cad7ab4b9887ce0b78eabd8/docker/run.sh).